### DISCUSSION TITLE
List of asterism names ...

### DISCUSSION THREAD

**Discussion by StellariumSentinel** [Stellarium/stellarium#3547] [General]
Good evening everyone, 
I would like to use asterisms on version 23.3 of stellarium, but at the moment I have only managed to identify the names of the winter and summer triangles (which work!):

AsterismMgr.selectAsterism("Winter Triangle");
AsterismMgr.selectAsterism("Summer Triangle");

Is there a complete list of asterism names available? I was trying to launch the "Big Dipper" but the name doesn't work. I also tried "Plow" or "Plow" but it doesn't work. Can someone help me!

Thankyou very very much!

Alby

**Comment by gzotti** [comment]
The structure of skycultures is described in the Stellarium User Guide

**Reply by StellariumSentinel** [reply]
Thank you Georg! 
I found it at this link: C:\Program Files\Stellarium\skycultures\modern\asterism_names.eng.fab
